styles/style.css

```diff
@@ -0,0 +1,34 @@
+nav{
```

Comment:
I would take a look at this example from W3 schools. It shows a basic html template with css. It shows you a header, body, and footer with styling. https://www.w3schools.com/w3css/tryit.asp?filename=tryw3css_containers_semantic_article